### PR TITLE
Updates to playbook with annotaton documentation

### PR DESCRIPTION
**Pull request by jperry18999**
image added
notes added
links updated

### CODE REVIEW

File: docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    - Test
+---
+
+## Introduction
+
+This document outlines best practices to follow when writing Playwright test scripts.
+
+## Locating selectors
+
+When planning testing at the readying phase of a work item, locators must be considered.  This allows Software Engineers to plan any additional work needed in order to enable our best practices.
```
Comment (KarenFarrell36) [inline docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md:19]: Would prefer to use the term Engineers here as there are cases where Test Engineers can create selectors.

File: docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+- page.getByRole() to locate by explicit and implicit accessibility attributes.
+- page.getByText() to locate by text content.
+- page.getByLabel() to locate a form control by associated label's text.
+- page.getByPlaceholder() to locate an input by placeholder.
+- page.getByAltText() to locate an element, usually image, by its text alternative.
+- page.getByTitle() to locate an element by its title attribute.
+
+
+If an element cannot be located by any of these, then we look for data-test IDs (for more info click [here](https://playwright.dev../../locators#locate-by-test-id)).  **Software Engineers will support creating the data-test-IDs where needed.**
```
Comment (KarenFarrell36) [inline docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md:35]: Some Test Engineers can create these too so think we should update this text

File: docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+- Use `test.only` to run a single test. (Playwright will only run the test with the `test.only` annotation applied to it).
+
+This should be used in instances where we want to focus on a single test and ignore all other tests. 
+
+**We should **NEVER** merge tests with `test.only` annotations into the main branch, but it has it's uses when executing tests locally or on work / feature branches at times.**
+
+- Use `test.fixme` to mark a test that is known to be failing and requires a fix. (Playwright will not run the test and it will be skipped).
+
+This should be used in instances where the test logic is fine but the test is failing due to a known defect. There should be a BLI in place to fix this within the next 2 sprints and we are aware a fix is in the works. Its imperative we include a comment above the `test.fixme` annotation to explain why the test is failing, along with a reference ID of the DevOps BLI.
```
Comment (KarenFarrell36) [inline docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md:72]: I guess the halo id could be used if BLI not yet created?

File: docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+**We can merge tests with `test.fixme` annotations into the main branch, but only temporarily. This should then be reverted back to `test` once the test is fixed.**
+
+- Use `test.skip` to skip a test. (Playwright will not run the test and it will be skipped).
+
+There are 2 main instances why we would use this annotation:
+
+1 - To skip a test temporarily in order to focus on other tests. This can be beneficial when running tests *locally* or on work / feature branches, but shouldnt be merged to main in these instances. 
+
+2 - To skip a work in progress test, whereby the logic in the test would fail at this particular instance in time because it's not ready for testing yet, but we want to keep the test in the codebase for future reference or enhancement.
```
Comment (KarenFarrell36) [inline docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md:82]: Should we mention this is rare as we wouldn't want to merge incomplete work as an every day practice?

File: docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+3 - To skip a test under certain conditions, such as not running the test on a specific browser if it is flaky. For example:
+
+            import { test, expect } from '@playwright/test';
+
+            test('Safari-only test', async ({ page, browserName }) => {
+            test.skip(browserName !== 'webkit', 'This feature is Safari-only');
+            // ...
+            });
+
+4 - To skip a failing test, under instances when we do not have a BLI in place to fix the test and we are unsure of the timframe of the fix. In the instance of a failing test with a known fix in the works, we should use `test.fixme` instead, explained further below.
```
Comment (KarenFarrell36) [inline docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md:93]: Spelling error - timeframe

Comment (KarenFarrell36) [reply docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md:93]: Is it current practice to have BLIs for a broken test?

File: docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md (hunk 2 above)
Comment (jperry18999) [reply docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md:35]: updated :)

File: docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md (hunk 3 above)
Comment (jperry18999) [reply docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md:72]: updated :)

File: docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md (hunk 4 above)
Comment (jperry18999) [reply docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md:82]: updated :)

File: docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md (hunk 5 above)
Comment (jperry18999) [reply docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md:93]: spelling error updated :)

for a skipped test that is failing, then we should have a BLI in place for traceability..... either to be actioned within 2 sprints (test.fixme) which should be on a delivery teams sprint board or on our minmi board (test.skip) when we dont know the timeframe of the fix. have updated the notes.

**Pull request merged**